src/store/rule/actions.ts

```diff
@@ -15,14 +15,17 @@ const actions = {
       'SAVE_RULE_FAILURE',
   )<RequestPayload<RuleToSave>, CreateResponse<Rule | null>, CreateResponse<Rule | null>>(),
   setSavedRule: createAction('SET_CREATE_RULE')<CreateResponse<Rule | null>>(),
+  resetSavedRule: createAction('RESET_CREATE_RULE')<null>(),
```

Comment:
Codacy has a fix for the issue: [Replace `'RESET_CREATE_RULE'` with `"RESET_CREATE_RULE"`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  resetSavedRule: createAction("RESET_CREATE_RULE")<null>(),
```